US-CBP/cbp-theme
# Ds-css build updates - Bulma SASS customization (starting with Colors, etc.)

#### What's this PR do?
This primarily makes customizations to some of Bulma's general and global SASS variables to match what is needed to math up with the CBP-Theme standards.

#### Where should the reviewer start?
Can start with looking at the initial customization done at the top of the /src/styles/main.scss file.

#### How should this be manually tested?
Can run few steps below:

1. Nuke Node_Modules
rm -rf node_modules

2. Reinstall build
npm i

3. Create dev build
npm run dev

4. Checkout demo index page 
npm run start

5. Then you can check the final CSS provided to see if this build is using correct Colors that were customized as the top of the /src/styles/main.scss file.

#### Any background context you want to provide?
We are customizing Bulma's (initial-variables & derived-variables) SASS to match with the CBP-Theme's primary SASS values where possible.

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
- Is there a blog post?
n/a

- Does the knowledge base need an update?
n/a
